demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -2461,47 +2458,63 @@ def convert_entity_types_to_id_set_headers(excluded_items_by_type: dict):
         "script": "scripts",
         "playbook": "playbooks",
         "classifier": "Classifiers",
-        "incidentfield": "IncidentField",
+        "incidentfield": "IncidentFields",
         "incidenttype": "IncidentTypes",
         "indicatorfield": "IndicatorFields",
         "indicatortype": "IndicatorTypes",
         "mapper": "Mappers",
         "dashboard": "Dashboards",
         "widget": "Widgets",
         "list": "Lists",
-        "report": "Reports"
+        "report": "Reports",
+        "layout": "Layouts"
     }
 
     for key in entity_type_to_header:
         if key in excluded_items_by_type:
             excluded_items_by_type[entity_type_to_header[key]] = excluded_items_by_type.pop(key)
 
 
-def remove_items_from_packs_section(id_set: dict, excluded_items: dict) -> None:
+def remove_items_from_packs_section(id_set: dict, excluded_items_by_pack: dict) -> None:
     """
     Given the excluded items dict, remove the items from the 'ContentItems' entry of the relevant pack in the id set.
 
     Args:
         id_set: id_set to remove entities from
-        excluded_items: a dictionary of items that need to be excluded from the id_set, aggregated by packs.
+        excluded_items_by_pack: a dictionary of items that need to be excluded from the id_set, aggregated by packs.
 
     """
 
     packs_section_from_id_set = id_set["Packs"]
-    for pack, pack_items in excluded_items.items():
+    for pack, pack_items in excluded_items_by_pack.items():
         pack_content_items = packs_section_from_id_set.get(pack, {}).get("ContentItems")
 
         if not pack_content_items:  # This pack has been excluded from the id_set
             continue
 
         for item_type, item_name in pack_items:
-            pack_content_items.get(f'{item_type}s', {}).remove(item_name)
+            item_type = item_type_to_content_items_header(item_type)
```

Comment:
is that similar to this mapping?
https://github.com/demisto/demisto-sdk/pull/1832/files#diff-555c51fe59132215b77801a3bb499d421ebd91468c9d7806cedced34215a3048R1377

Reply:
very close but no :( 
For example = "IncidentTypes" != "incidentTypes"
But it could help me with another mapping, will be used